mobile_app_connector/models/app_hub.py

```diff
@@ -50,7 +50,13 @@ def mobile_get_message(self, partner_id, **pagination):
         #  is correspondent (to avoid viewing letters when he doesn't write)
         sponsorships = (partner.contracts_correspondant +
                         partner.contracts_fully_managed).filtered('is_active')
+        unpaid = (partner.contracts_correspondant +
```

Comment:
I would only take `contracts_fully_managed`, because we don't want to invite him to pay the sponsorship if he isn't the person who pays.

Reply:
Done. Thank you !